userguide/sharing.rst

```diff
@@ -318,8 +318,16 @@ Creating Authenticated and Time Machine Shares
 macOS includes the Time Machine feature which can be used to perform
 automatic back ups. In this configuration example, a Time Machine user
 is configured to back up to an AFP share on a %brand% system. The
-process for creating an authenticated share for a user is the same as
-creating a Time Machine share for that user.
+process for creating an authenticated share for a user is the same
+as creating a Time Machine share for that user.
+
+.. note:: An :ref:`SMB share <Windows (SMB) Shares>` can be used
+   instead of an AFP share for Time Machine user backups. Follow this
```

Comment:
Which process is "this" process?  The one I am reading, or the one in the link?

We need an SMB-specific Time Machine section that can be expanded and updated over time.  The AFP version will probably not change, and the two are likely to differ eventually.